### ISSUE TITLE
[Accessibility][Teaser][Quick Search] Image links redundant, some lack text

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1682]
**Steps to Reproduce**
1. With NVDA operating, open the AEM Core Components Component Library landing page (https://www.aemcomponents.dev/content/core-components-examples/library.html) 
1. Navigate through search results on the page using the Tab key. Note duplicate tab stops and link announcements for each search result.
1. Navigate to "Product List" search result item. Note lack of appropriate textual alternative for image link.

**Current Behavior**
Each search result item in the list of search results has two duplicate links to the same destination. The first link is the linked thumbnail image, and the second link is the text of the heading for the search result item. This results in two tab stops and repetitive link text when navigating using a screen reader.

Each image that has alt text also has a `title` attribute, resulting in further repetitive text.

For example, navigating to the Breadcrumb component in a list of search results with NVDA, the first tab stop is announced as "Breadcrumb  graphic    Breadcrumb  link". The second tab stop is announced as "Breadcrumb  link    heading    level 2".

Some linked images do not have `alt` or `title` attributes. As a result, the filename is announced as the link text. In some cases the filename is an appropriate (e.g. when it is a single word, like "product") but in other cases the screen reader is unable to pronounce the filename as meaningful text (e.g. "productlist").

**Expected behavior/code**
Link text should be descriptive. Ensure that image links provide appropriate textual alternatives.  Ideally each search result would be only one tab stop: a single link with descriptive text. 

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users, keyboard users

**WCAG SC**
2.4.4 Link Text (In Context)
1.1.1 Non-text Content

**Possible Solution**
Consider removing the `title` attribute from image links that have alt text, to avoid duplicate announcements.

Consider whether the link can be removed from the image entirely. If the intention is for the whole card to be clickable, consider using javascript to handle a mouse click on the card while leaving only the heading text linked.

**Comment by bpauli** [comment]
@adobe export issue to Jira project SITES

**Comment by gabrielwalt** [comment]
The lack of text information on the "Product List" image is specific to the sample content of the component library, and not a core component bug.
The rest of the double-tab issues will be addressed by #1712.
